setup/www/resources/config/nodejs.org

```diff
@@ -219,11 +236,26 @@ server {
     location / {
         rewrite ^/$ /en/ redirect;
 
+        try_files $uri $uri/ @english_fallback;
+
         location ~ \.json$ {
             add_header access-control-allow-origin *;
         }
     }
 
+    # instead of serving a 404 page when a page hasn't been translated
+    location @english_fallback {
+        if ($uri ~* ^/(it|ko)/) {
+            set $lang $1;
+        }
+        rewrite ^/(it|ko)/(.*)$ /en/$2;
```

Comment:
@jbergstroem do you think relative redirect paths here are okey, or should we use absolute URLs?